scripts/globals/abilities/chi_blast.lua

```diff
@@ -29,4 +29,8 @@ function onUseAbility(player,target,ability)
     target:updateEnmityFromDamage(player,dmg)
     player:delStatusEffect(dsp.effect.BOOST)
     return dmg
+
+	if dmg > 0 and target(isAsleep) then
```

Comment:
the return above means this will never be hit.

Reply:
Woops, it's early plz forgive